orix/quaternion/orientation.py

```diff
@@ -375,6 +375,73 @@ def get_distance_matrix(
 
         return angles
 
+    @classmethod
+    def from_align_vectors(
+        cls,
+        to_: Miller,
+        from_: Miller,
+        weights: Optional[np.ndarray] = None,
+        return_rmsd: bool = False,
+        return_sensitivity: bool = False,
+    ) -> Misorientation:
+        """ "Return an estimated misorientation to optimally align two sets of vectors, one in each crystal."
+
+        This method wraps :meth:`scipy.spatial.transform.Rotation.align_vectors`,
+        see that method for further explanations of parameters and returns.
+
+        Parameters
+        ----------
+        to_
+            Directions in the other crystal.
+        from_
+            Directions in the initial crystal.
+        weights
+            The relative importance of the different vectors.
+        return_rmsd
+            Whether to return the root mean square distance (weighted)
+            between ``to_`` and ``from_`` after alignment.
+        return_sensitivity
+            Whether to return the sensitivity matrix.
+
+        Returns
+        -------
+        estimated misorientation
+            Best estimate of the ``misorientation`` that transforms ``from_`` to ``to_``.
+            The symmetry of the ``misorientation`` is inferred from the phase of ``to_`` and ``from_``, if given.
+        rmsd
+            Returned when ``return_rmsd=True``.
+        sensitivity
+            Returned when ``return_sensitivity=True``.
+
+        Examples
+        -------
```

Comment:
Sphinx is nitpicky, the single missing "-" makes the example render incorrectly ([see docs built from this PR](https://orix--401.org.readthedocs.build/en/401/reference/generated/orix.quaternion.Misorientation.from_align_vectors.html)).

```suggestion
        --------
```

I recommend to check formatting locally before pushing by following our [contributing guide](https://orix.readthedocs.io/en/stable/contributing.html#build-and-write-documentation).